Modules/vc-module-pricing/VirtoCommerce.PricingModule.Web/Localizations/en.VirtoCommerce.Pricing.json

```diff
@@ -246,5 +246,14 @@
         "VirtoCommerce.Pricing": {
             "description": "Export/Import price lists, assignments and prices"
         }
+    },
+    "export.type-names": {
```

Comment:
please make structured object instead of names with dots:
```
 "export": {
   "type-names": {        
       …
    },
    "type-groups": {
        "Pricing": "Pricing"
    }	   
 }
```